src/pages/settings/Security/TwoFactorAuth/IsEnabledPage.js

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react';
+import {Text, View} from 'react-native';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import ROUTES from '../../../../ROUTES';
+import Section from '../../../../components/Section';
+import * as Illustrations from '../../../../components/Icon/Illustrations';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import themeColors from '../../../../styles/themes/default';
+import styles from '../../../../styles/styles';
+import ConfirmModal from '../../../../components/ConfirmModal';
+
+const propTypes = {
+    ...withLocalizePropTypes,
```

Comment:
Maybe it is better to just assign `withLocalizePropTypes` direct to `IsEnabledPage.defaultProps`